community/modules/scripts/ramble-setup/templates/ramble_setup.yml.tpl

```diff
@@ -36,32 +22,38 @@
     chown_owner: ${chown_owner}
     chgrp_group: ${chgrp_group}
   tasks:
+  - name: Acquire lock
+    ansible.builtin.shell:
+      mkdir -p "{{ install_dir | dirname }}/.ramble_lock"
```

Comment:
I believe the `-p` will basically never fail. My recommendation would be to have a first task that creates install_dir using a -p, then a second task that creates .ramble_lock without the -p and use this second task to capture the return behavior with the lock.